pkg/server/server_test.go

```diff
@@ -1981,3 +1981,180 @@ func TestIsExperimentallyEnabled(t *testing.T) {
 		require.False(t, s.IsExperimentallyEnabled(someExperimentalFlag))
 	})
 }
+
+func TestErrorThrownIfConsistencyRequestedWithoutFlagEnabled(t *testing.T) {
+	ds := memory.New()
+	openfga := MustNewServerWithOpts(WithDatastore(ds))
+
+	t.Run("check_throws_error_if_higher_consistency_requested_without_flag", func(t *testing.T) {
```

Comment:
```suggestion
func TestErrorThrownIfConsistencyRequestedWithoutFlagEnabled(t *testing.T) {
        t.Cleanup(func() {
		goleak.VerifyNone(t)
	})
	ds := memory.New()
	t.Cleanup(ds.Close)
	openfga := MustNewServerWithOpts(WithDatastore(ds))
	t.Cleanup(openfga.Stop)

	t.Run("check_throws_error_if_higher_consistency_requested_without_flag", func(t *testing.T) {
```

Reply:
Done in c3c6d511e255f424d3c85e7a61f9e376cbc5f478, I had to add some `Close` calls in the `TestIsExperimentallyEnabled` test in order to get this goleak check happy